wgpu-core/src/binding_model.rs

```diff
@@ -39,6 +39,8 @@ pub enum BindGroupLayoutEntryError {
     StorageTextureCube,
     #[error("Read-write and read-only storage textures are not allowed by webgpu, they require the native only feature TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES")]
     StorageTextureReadWrite,
+    #[error("Atomic storage textures are not allowed by webgpu, they require the native only feature TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES")]
```

Comment:
```suggestion
    #[error("Atomic storage textures are not allowed by baseline webgpu, they require the native only feature TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES")]
```

I'm not sure what the exact wording should be, but this feels off, notably around "not allowed by webgpu" - we don't really present to the users the fact that we're webgpu - this isn't allowed not because webgpu doesn't allow it, but because it's missing this feature.

I know the previous error message is also like that, and probably both need reworking.